Library/Homebrew/cmd/install.rb

```diff
@@ -123,11 +131,10 @@ def check_writable_install_location
   def check_xcode
     checks = Checks.new
     %w[
-      check_for_unsupported_osx
```

Comment:
No, I think that was a mistake during one of my rebases. Should I put it under `check_xcode` or something more general, seeing how `check_xcode` is only executed if Xcode is installed?